plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityArrowPierceLevel.java

```diff
@@ -0,0 +1,42 @@
+package com.denizenscript.denizen.objects.properties.entity;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import org.bukkit.entity.AbstractArrow;
+
+public class EntityArrowPierceLevel extends EntityProperty<ElementTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name pierce_level
+    // @input ElementTag(Number)
+    // @description
+    // Controls the number of times this arrow can pierce through an entity. Must be between 0 and 127 times.

```

Comment:
Modern property meta is generally just what the property _is_, e.g. `The number of times an arrow will pierce through entities` or something like that.
Also I feel like what it actually does would be clearer with something like `The number of entities an arrow will pierce through while flying.`, currently it sounds a bit like it might be hitting a single entity multiple times.